src/test/java/core/basesyntax/services/DataFileReaderTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.services;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataFileReaderTest{
+    private static final String FILE_PATH
```

Comment:
```suggestion
    private static final String FILE_PATH = "src/test/resources/testFile.csv";
```
try to inline declaration and initialization if possible

Reply:
Done where it was possible